gcloud/bigtable/client.py

```diff
@@ -27,26 +27,39 @@
 """
 
 
+from gcloud.bigtable._generated import bigtable_cluster_service_pb2
+from gcloud.bigtable._generated import bigtable_service_pb2
+from gcloud.bigtable._generated import bigtable_table_service_pb2
+from gcloud.bigtable._generated import operations_pb2
+from gcloud.bigtable._helpers import make_stub
 from gcloud.client import _ClientFactoryMixin
 from gcloud.client import _ClientProjectMixin
 from gcloud.credentials import get_credentials
 
 
+TABLE_STUB_FACTORY = (bigtable_table_service_pb2.
+                      early_adopter_create_BigtableTableService_stub)
```

Comment:
Tell me about it. I asked @nathanielmanistaatgoogle nicely about it a while ago, but he said it didn't matter because alpha was becoming beta soon.

On that note, this will change once I upgrade the library to use the beta version.